src/main.cpp

```diff
@@ -3477,6 +3477,7 @@ bool static ConnectTip(CValidationState& state, const CChainParams& chainparams,
 
```

Comment:
Should we update the copyright at the top of the file? 2017-2018?

Reply:
go for it